apps/client/src/routes.js

```diff
@@ -30,61 +20,79 @@ export const routeConfig = [
     name: "home",
     exact: true,
     path: "/",
-    component: process.env.NODE_ENV !== "production" && DevHomePage,
+    component:
+      process.env.NODE_ENV !== "production" &&
+      React.lazy(() => import(`./pages/DevHomePage`)),
```

Comment:
We _could_ refactor this to a separate function `loadLazyFile = (file) => React.lazy(() => import(file))`

Reply:
We _could_ do that, however there's a feature in webpack we use, see `/* webpackPrefetch: true */` comments. Wouldn't want to break that.